private/buf/bufcli/flags_args.go

```diff
@@ -148,8 +148,8 @@ func BindCreateVisibility(flagSet *pflag.FlagSet, addr *string, flagName string,
 	flagSet.StringVar(
 		addr,
 		flagName,
-		"",
-		fmt.Sprintf(`The repository's visibility setting, if created. Must be set if --%s is set. Must be one of %s`, createFlagName, stringutil.SliceToString(allVisibiltyStrings)),
+		privateVisibility,
+		fmt.Sprintf(`The repository's visibility setting, if created. Can only be set if --%s is set. Defaults to %s if this flag is not set with --%s. Must be one of %s`, createFlagName, privateVisibility, createFlagName, stringutil.SliceToString(allVisibiltyStrings)),
```

Comment:
```suggestion
		fmt.Sprintf(`The repository's visibility setting, if created. Can only be set with --%s. Must be one of %s`, createFlagName, stringutil.SliceToString(allVisibiltyStrings)),
```

This will replace:
```
The repository's visibility setting, if created. Can only be set if --create is set. Defaults to private if this flag is not set with --create. Must be one of [public,private] (default "private")
```
with:
```
The repository's visibility setting, if created. Can only be set with --create. Must be one of [public,private] (default "private")
```